api/base/parsers.py

```diff
@@ -119,3 +119,6 @@ class JSONAPIParserForRegularJSON(JSONAPIParser):
     Allows same processing as JSONAPIParser to occur for requests with application/json media type.
     """
     media_type = 'application/json'
+
+class JSONAPIRelationshipParser(JSONParser):
```

Comment:
I think the JSONAPIRelationshipParser needs its own parse method, even though the data is simple.  We use the JSONAPIParser parse method to do many things.  It checks the formatting to ensure it matches the jsonapi spec and then only returns the important properties.  Relationship data is going to come in looking like this:
```
{
    "data": {
          "id": "UVA",
          "type": "institution"
     }
}
```
The parser can strip off the "data" key; it isn't needed.  It should also validate that "id" and "type" are present.